src/shared/components/palette/history/event-log/event-log-table.component.spec.ts

```diff
@@ -108,7 +108,8 @@ describe('EventLogTableComponent', () => {
       expect(firstRowCells.length).toBe(3);
       let firstEvent = EVENTS[0];
 
-      const timeZoneOffset = -(new Date().getTimezoneOffset());
+      const date = new Date(2017, 4, 10); // 10th May, 2017
```

Comment:
What is the significance of specifying 10th May 2017 and not using the current date?

Reply:
I would presume nothing, it's just a test right?

Reply:
Its code that was previously there and from what I gather it needs to be a date that falls in the summer otherwise the test will fail (hence the fix).

Reply:
If you change a test, it's usually because it wasn't passing without the change. I'm just checking that this change isn't simply to get through a failing test without addressing the underlying failure. :)

Reply:
> Its code that was previously there and from what I gather it needs to be a date that falls in the summer otherwise the test will fail (hence the fix).

Cool cool cool. Then have yourself an approval.

Reply:
Actually @paul-graham is correct. I believe this test only fails locally. In fact, I believe it will fail on the server because of this change.